appveyor.yml

```diff
@@ -15,14 +15,13 @@ build:
   parallel: true
   verbosity: minimal
 deploy:
-  release: tgstation-server-3-v$(appveyor_build_version)
+  release: tgstation-server-v$(appveyor_build_version)
   description: 'The /tg/station server suite'
   provider: GitHub
   auth_token:
-    secure: github_oath_token
+    secure: 9Yf+FtoE7nf36V3dyVRANkNZ7og4dp/0uojCS51vrvIeNOTIM8Nq24lylgVBOhSL

```

Comment:
Can't be and still be useful.

Encrypted how?

Decrypted how?

What is it used for?

At the end of the day either somebody could still abuse this and auth themselves with the token, or the token is useless.